src/RoomClient.ts

```diff
@@ -216,6 +231,14 @@ class RoomClient {
     this.emitter.off(name, handler);
   }
 
+  get ConnectionState(): ConnectionState {
```

Comment:
camelCase convention for a property:

```suggestion
  get connectionState(): ConnectionState {
```